docs/docs/11-advanced-performance.md

```diff
@@ -100,7 +100,7 @@ shouldComponentUpdate: function(nextProps, nextState) {
 }
 ```
 
-Basically, we ended up doing a deep comparison to make sure we properly track changes. This approach is pretty expensive in terms of performance and it doesn't scale as we would have to write different deep equality code for each model. On top of that, it might not even work if we don't carefully manage object references. Say this component is used by a parent:
+Basically, we ended up doing a deep comparison to make sure we properly track changes. This approach is pretty expensive, in terms of performance, and it doesn't scale as we would have to write different deep equality code for each model. On top of that, it might not even work if we don't carefully manage object references. Say this component is used by a parent:
```

Comment:
> This approach is pretty expensive, in terms of performance, and it doesn't scale as we would have to write different deep equality code for each model.

I actually read this sentence much more like it was originally written, though I do pause at the "and". Let's lost that first comma.